### PR TITLE
[ops]feat: add mlu liger_kernel support 

### PR DESCRIPTION
**Pull request by sc1915**
### What does this PR do?

> Add liger kernel support on Cambricon MLU.

### API and Usage Example

> No new public API. Existing selection controls the fast path:
```
model:
  ops_implementation:
    rms_norm_implementation: liger_kernel
```

### What's in the change

- add kernel: cross_entropy_loss, rms_norm, rotary_pos_emb, swiglu_mlp implementation in Liger-Kernel for improved perfermence on Cambricon MLU.

## Summary by CodeRabbit

* **New Features**
  * Added Cambricon MLU support for Liger Kernel implementations of RMSNorm, rotary positional embeddings, SwiGLU MLP, and cross-entropy loss.
  * These implementations can now be selected and used on supported MLU hardware instead of automatically falling back to eager execution.

### CODE REVIEW

File: veomni/arguments/arguments_types.py [hunk 1]
```diff
@@ -1172,15 +1172,15 @@ def _resolve_profile(self):
 
 # MLU compatibility tables for ``_validate_implementations``.
 _MLU_ALLOWED: Dict[str, frozenset] = {
+    "rms_norm_implementation": frozenset({"liger_kernel"}),
```
Comment (FoolPlayer) [inline veomni/arguments/arguments_types.py:1175]: Is there any liger-kernel version requirements for mlu use?
